functional-tests/tests/version_test.py

```diff
@@ -9,4 +9,4 @@ def test_version_output():
         logfile=sys.stdout
     )
 
-    process.expect('@smartthings/cli/.+ .+ node-.+')
+    process.expect('^\\d+\\.\\d+\\.\\d+')
```

Comment:
I'm having trouble figuring out what this change is.  Is this because it's not a monorepo anymore?